### PR TITLE
Add Joël Porquet-Lupine to UC Davis

### PR DESCRIPTION
**Pull request by joel-porquet**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-j.csv [hunk 1]
```diff
@@ -1461,6 +1461,7 @@ Joël M. H. Karel,Maastricht University,https://dke.maastrichtuniversity.nl/joel
 Joel Moses,Massachusetts Institute of Technology,https://esd.mit.edu/Faculty_Pages/moses/moses.htm,NOSCHOLARPAGE
 Joël Ouaknine [SWS],Max Planck Society,https://people.mpi-sws.org/~joel,ZfE4iQ4AAAAJ
 Joel P. Ilao,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/faculty-profile/single/?id=32742738160&command=GETPROFILE,gd8W9ecAAAAJ
+Joël Porquet-Lupine,Univ. of California - Davis,https://cs.ucdavis.edu/directory/joel-porquet-lupine,hyHs8joAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv:1464]: Please clarify how this faculty member meets the inclusion criteria (tenure-track). I know that at least one UC school (UCSD) has a tenure track for teaching faculty; please verify that this is the case for Davis. Thanks.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-j.csv (hunk 1 above)
Comment (joel-porquet) [reply csrankings-j.csv:1464]: Hi, the tenure-track teaching faculty series exists across all UC campuses. UCSD (and UCB) may have been a little bit ahead some years ago but UC Davis now has 50+ professors of teaching on our campus.

Here is a document from UC Riverside explaining the professor of teaching series: https://efileinfo.ucr.edu/sites/default/files/2023-02/ucr-guidelines-for-professor-of-teaching-series_7.27.21.pdf. It mentions that *"Professor of Teaching titles are **tenure-track positions** designed to meet the long-term instructional needs of the
University that cannot be best fulfilled by an appointee in the regular professorial series."*

Hope this clarifies your concerns.

Comment (emeryberger) [reply csrankings-j.csv:1464]: Thanks. Missing piece: can you solely advise PhD students in CS? This is a requirement.

**Comment by joel-porquet** [comment]
I don't know of a document that clearly indicates that, but as far as I
know, professors of teaching can be sole advisors of PhD students and can
be sole PIs on grants. We are members of the academic senate.

**Comment by emeryberger** [comment]
Membership in the academic senate doesn't tell me much (our faculty senate, for example, comprises non-TT faculty among others). I need unequivocal confirmation that you (and those in a similar role) can solely advise PhD students in CS.

**Comment by joel-porquet** [comment]
I found this page that explains who can mentor grad students at UC Davis
and be "Major professors":
https://grad.ucdavis.edu/graduate-program-roles-and-responsibilities

In the section "Mentor" (the last one), it says the following:

---
In the UC system, only faculty can mentor graduate students. At other
campuses, this typically means the Academic Senate. However, UC Davis
defines faculty (for the purpose of mentoring graduate students) more
broadly than the Academic Senate. The categories of mentor eligibility are
outlined in the Service on Advanced Degree Committees policy:

- members of the Academic Senate <http://academicsenate.ucdavis.edu/> of
the University of California (including professors, lecturers, senior
lecturers with security of employment, professors in residence, professors
of clinical “__,” acting professor series);
- [...]

That is, by order of Graduate Council, meeting one of these requirements is
necessary in order to serve as a Major Professor (i.e. Chair of a thesis or
dissertation committee). [...]
---

FYI, the professor of teaching series is the working title of the "Lecturer
with (potential) security of employment" series. So we belong to the first
bullet point above.

This should hopefully address your concerns.

**Pull request merged**